[2022-10-01 19:45:17,167] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:44:19.977775+00:00 [queued]>
[2022-10-01 19:45:17,184] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:44:19.977775+00:00 [queued]>
[2022-10-01 19:45:17,184] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:45:17,185] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-01 19:45:17,185] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-01 19:45:17,211] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_fct_table> on 2022-10-01 19:44:19.977775+00:00
[2022-10-01 19:45:17,220] {standard_task_runner.py:52} INFO - Started process 394 to run task
[2022-10-01 19:45:17,224] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'store_fct_table', 'manual__2022-10-01T19:44:19.977775+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp8_lu6ayn', '--error-file', '/tmp/tmptql_eq0y']
[2022-10-01 19:45:17,226] {standard_task_runner.py:80} INFO - Job 54: Subtask store_fct_table
[2022-10-01 19:45:17,310] {task_command.py:369} INFO - Running <TaskInstance: main_dag.store_fct_table manual__2022-10-01T19:44:19.977775+00:00 [running]> on host cb37a969dddb
[2022-10-01 19:45:17,426] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=store_fct_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-01T19:44:19.977775+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-01T19:44:19.977775+00:00
[2022-10-01 19:45:17,429] {postgres.py:127} INFO - Running copy expert: COPY fct_table FROM stdin WITH DELIMITER as ',', filename: df_fct_table.csv
[2022-10-01 19:45:17,446] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-01 19:45:17,463] {python.py:173} INFO - Done. Returned value was: None
[2022-10-01 19:45:17,480] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=store_fct_table, execution_date=20221001T194419, start_date=20221001T194517, end_date=20221001T194517
[2022-10-01 19:45:17,519] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-01 19:45:17,595] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
